src/Middleware/RateLimiting/src/Features/IRateLimiterStatisticsFeature.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading.RateLimiting;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.RateLimiting.Features;
```

Comment:
No need to put this in its own namespace and folder. It just causes extra usings elsewhere.

I'll double check with the API review.
https://github.com/dotnet/aspnetcore/issues/45658#issuecomment-1381032567